test/e2e/multihoming_utils.go

```diff
@@ -168,6 +168,21 @@ func generatePodSpec(config podConfiguration) *v1.Pod {
 	if config.isPrivileged {
 		privileged := true
 		podSpec.Spec.Containers[0].SecurityContext.Privileged = &privileged
+	} else {
+		falsePtr := false
+		truePtr := true
+		var userID int64 = 1000
+		for _, container := range podSpec.Spec.Containers {
+			if container.SecurityContext.Capabilities == nil {
+				container.SecurityContext.Capabilities = &v1.Capabilities{}
+			}
+			container.SecurityContext.Capabilities.Drop = []v1.Capability{"ALL"}
+			container.SecurityContext.Privileged = &falsePtr
```

Comment:
nit:
"k8s.io/utils/ptr"
gives this nice `ptr.To(true)` and `ptr.To(false)`
utils which can avoid needed those extra variables